src/graphql/types/scalar/level.ts

```diff
@@ -0,0 +1,25 @@
+import { GT } from "@graphql/index"
+
+const AccountLevel = GT.Enum({
+  name: "AccountLevel",
+  values: {
+    ZERO: {},
+    ONE: {},
+    TWO: {},
```

Comment:
shouldnt we use AccountLevel enum here ? (check src/domain/accounts/primitives.ts)